ledger/src/bigtable_upload.rs

```diff
@@ -186,7 +196,12 @@ pub async fn upload_confirmed_blocks(
                 num_blocks -= 1;
                 None
             }
-            Some(confirmed_block) => Some(bigtable.upload_confirmed_block(slot, confirmed_block)),
+            Some(confirmed_block) => {
+                let bt = bigtable.clone();
+                Some(tokio::spawn(async move {
+                    bt.upload_confirmed_block(slot, confirmed_block).await
```

Comment:
would be nice if there was a tokio receiver that let you clone the receiver and use multiple tasks to pop blocks off the queue and run in parallel; this thing could rip!